processes/record_frbrizer.py

```diff
@@ -0,0 +1,101 @@
+from lxml import etree
+import re
+
+from logger import create_log
+from mappings.oclc_bib import OCLCBibMapping
+from mappings.oclcCatalog import CatalogMapping
+from managers import DBManager, OCLCCatalogManager, RedisManager
+from model import Record
+from .record_buffer import RecordBuffer
+
+
+logger = create_log(__name__)
+
+
+class RecordFRBRizer:
+
+    def __init__(self, db_manager: DBManager):
+        self.db_manager = db_manager
+
+        self.oclc_catalog_manager = OCLCCatalogManager()
+        
+        self.redis_manager = RedisManager()
```

Comment:
last question - is there a reason these are constructed here, while the db manager is passed in? My gut is to just do DI and pass everything in, but I also don't have a great handle on this codebase 😆

Reply:
good question! we can but I think this might be the only step in the process that requires the redis_manager!